VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json

```diff
@@ -1060,8 +1060,12 @@
   "quick-participant-landing-page": {
     "page-title": "Enter your details to join your hearing",
     "paragraph1": "Please tell us your name and your role in the hearing.",
+    "problem-with-form": "There is a problem",
+    "problem-with-special-char": "You entered some characters that are not allowed.",
+    "problem-with-special-char-recommendation": "You may use accents, apostrophes and hyphens but no other symbols.",
     "name-label": "Your full name",
-    "name-error": "Please enter your full name",
+    "name-error-empty": "Please enter your full name",
+    "name-error-special-char": "You've entered characters that are not allowed",
```

Comment:
Is there a ticket for Welsh translation?

Reply:
Lorenzo made a note to pass the content to be translated by the Welsh team

Reply:
..so no ticket at the moment

Reply:
Maybe ask Lorenzo to make a ticket, especially if he's going to be gone for a while.

Reply:
Maybe ask Lorenzo to make a ticket, especially if he's going to be gone for a while.